src/components/App.js

```diff
@@ -452,6 +468,8 @@ class App extends Component {
 
         await this.loadGameTrees([emptyTree], {suppressAskForSave})
 
+        try {logger.rotateLog()} catch (err) {}
```

Comment:
Why not call `rotateLog()` in `loadGameTrees()`? That way you don't need to call it in `newFile()`, `loadFile()`, and `loadContents()` separately.

Reply:
Hmm, good point. This is fixed in ee14a8a

Reply:
Oops I put it in the wrong place, 86147a5 should definitely fix it =)